---
title: C6322 | Microsoft Docs
ms.custom: ''
ms.date: 2018-06-30
ms.prod: visual-studio-dev14
ms.reviewer: ''
ms.suite: ''
ms.technology:
- vs-devops-test
ms.tgt_pltfrm: ''
ms.topic: article
f1_keywords:
- C6322
helpviewer_keywords:
- C6322
ms.assetid: fb23d2b1-b2a0-465c-8bf5-ec039c6c7757
caps.latest.revision: 15
author: corob-msft
ms.author: gewarren
manager: ghogen
ms.openlocfilehash: 2c96017e0090dec17a2bcccbed72620da99fe0b5
ms.sourcegitcommit: 55f7ce2d5d2e458e35c45787f1935b237ee5c9f8
ms.translationtype: MT
ms.contentlocale: ko-KR
ms.lasthandoff: 08/22/2018
ms.locfileid: "47556021"
---
# <a name="c6322"></a>C6322
[!INCLUDE[vs2017banner](../includes/vs2017banner.md)]

이 항목의 최신 버전에서 찾을 수 있습니다 [C6322](https://docs.microsoft.com/visualstudio/code-quality/c6322)합니다.  
  
경고 C6322: 빈 _except 블록  
  
 이 메시지는 _except 블록의 코드가 없는 임을 나타냅니다. 결과적으로, 예외 처리 되지 않은 상태가 될 수 있습니다.  
  
## <a name="example"></a>예제  
 다음 코드에서는이 경고를 생성합니다.  
  
```  
#include <stdio.h>  
#include <excpt.h>  
#include <windows.h>  
  
void fd(char *pch)  
{  
   __try  
     {  
       // exception rasied if pch is null  
       *pch= 0 ;  
     }  
   __except(GetExceptionCode() == EXCEPTION_ACCESS_VIOLATION)  
     {  
     }  
}  
```  
  
 이 경고를 해결 하려면 다음 코드를 사용 합니다.  
  
```  
#include <stdio.h>  
#include <excpt.h>  
#include <windows.h>  
  
void f(char *pch)  
{  
   __try  
     {  
       // exception rasied if pch is null  
      *pch= 0 ;  
     }  
   __except(GetExceptionCode() == EXCEPTION_ACCESS_VIOLATION ?   
               EXCEPTION_EXECUTE_HANDLER : EXCEPTION_CONTINUE_SEARCH)  
     {  
       // code to handle exception  
       puts("Exception Occurred");     
     }  
}    
```  
  
## <a name="see-also"></a>참고 항목  
 [try-except 문](http://msdn.microsoft.com/library/30d60071-ea49-4bfb-a8e6-7a420de66381)



